src/oneD/refine.cpp

```diff
@@ -77,39 +80,38 @@ int Refiner::analyze(size_t n, const double* z, const double* x)
             string name = m_domain->componentName(i);
             // get component i at all points
             for (size_t j = 0; j < n; j++) {
-                v[j] = value(x, i, j);
+                val[j] = value(x, i, j);
             }
 
             // slope of component i
             for (size_t j = 0; j < n-1; j++) {
-                s[j] = (value(x, i, j+1) - value(x, i, j))/(z[j+1] - z[j]);
+                slope[j] = (val[j+1] - val[j]) / dz[j];
             }
 
-            // find the range of values and slopes
-            double vmin = *min_element(v.begin(), v.end());
-            double vmax = *max_element(v.begin(), v.end());
-            double smin = *min_element(s.begin(), s.end());
-            double smax = *max_element(s.begin(), s.end());
-
-            // max absolute values of v and s
-            double aa = std::max(fabs(vmax), fabs(vmin));
-            double ss = std::max(fabs(smax), fabs(smin));
-
-            // refine based on component i only if the range of v is
-            // greater than a fraction 'min_range' of max |v|. This
-            // eliminates components that consist of small fluctuations
-            // on a constant background.
-            if ((vmax - vmin) > m_min_range*aa) {
-                // maximum allowable difference in value between adjacent
-                // points.
-                double dmax = m_slope*(vmax - vmin) + m_thresh;
+            // find the range of values and slopes of component i over the domain
+            double val_min = *min_element(val.begin(), val.end());
+            double val_max = *max_element(val.begin(), val.end());
+            double slope_min = *min_element(slope.begin(), slope.end());
+            double slope_max = *max_element(slope.begin(), slope.end());
```

Comment:
```suggestion
            double valMin = *min_element(val.begin(), val.end());
            double valMax = *max_element(val.begin(), val.end());
            double slopeMin = *min_element(slope.begin(), slope.end());
            double slopeMax = *max_element(slope.begin(), slope.end());
```
Per Cantera's C++ style guide. Same comment applies to other variable names below.